### PR TITLE
Bump org.owasp:dependency-check-maven to 6.2.2

### PR DESCRIPTION
**Pull request by mprins**
And update the config to suppress false positives in some of the community modules.

While here also remove a duplicate dependency declaration in `src/community/oseo/wb-oseo/pom.xml` that maven complains about.

  

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [x] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [ ] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [ ] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [ ] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [ ] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [x] Bug fixes and small new features are presented as a single commit.
- [ ] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

File: src/community/oseo/web-oseo/pom.xml [hunk 1]
```diff
@@ -61,13 +61,6 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
-    <dependency>
```
Comment (bradh) [inline src/community/oseo/web-oseo/pom.xml:64]: This part looks unrelated and possibly undesired.

**Review by bradh — COMMENTED** [review]
Generally looks OK except for the change in the web-oseo pom.

File: src/community/oseo/web-oseo/pom.xml (hunk 1 above)
Comment (mprins) [reply src/community/oseo/web-oseo/pom.xml:64]: when you look ~10 lines up you can see the duplicate declaration: https://github.com/geoserver/geoserver/blob/0ef63e17696b7b4c278e78b3138968101c16d0f9/src/community/oseo/web-oseo/pom.xml#L43-L49
